eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -603,13 +605,13 @@ jobs:
           env:
             SYSTEM_ACCESSTOKEN: $(System.AccessToken)
 
-    - ${{ if ne(parameters.runOnline, 'True' )}}:
-      - script: |
-          set -ex
-          # Update the owner of the staging directory to the current user
-          sudo chown -R $(whoami) $(Build.ArtifactStagingDirectory)
-        displayName: Update owner of artifacts staging directory
-
+    - ${{ if ne(parameters.runOnline, 'True' )}}:
+      - script: |
+          set -ex
+          # Update the owner of the staging directory to the current user
+          sudo chown -R $(whoami) $(Build.ArtifactStagingDirectory)
+        displayName: Update owner of artifacts staging directory

```

Comment:
this is unrelated but GitHub told me there are mixed line endings in the file and it would fix it